api/src/test/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimControllerTest.java

```diff
@@ -63,7 +65,7 @@
     })
     void shouldInvokeClaimServiceAndReturnCorrectStatusWithVoucherEntitlement(ClaimStatus claimStatus, HttpStatus httpStatus) {
         // Given
-        NewClaimDTO dto = aValidClaimDTOWithEligibilityOverrideOutcome(EligibilityOutcome.CONFIRMED);
+        NewClaimDTO dto = aValidClaimDTOWithEligibilityOverrideOutcome(EXPECTED_DELIVERY_DATE_IN_TWO_MONTHS, NO_CHILDREN, EligibilityOutcome.CONFIRMED);
```

Comment:
I'm not sure if I missed this in a previous PR, but the test name doesn't seem to represent what's going on, which is testing that an overridden confirmed decision should return the desired result

Reply:
That looks like we edited the wrong test method :(

Reply:
I've restored the original method and added a new one